packages/itwinui-react/src/core/ThemeProvider/ThemeProvider.tsx

```diff
@@ -40,6 +40,14 @@ export type ThemeOptions = {
    * Will default to user preference if browser supports it.
    */
   highContrast?: boolean;
+  /**
+   * If enabled, the theme resembles the future iTwinUI version's theme (including alphas) *whenever possible*.
+   *
+   * This is useful in making apps looks like future versions of iTwinUI to help with incremental adoption.
+   *
+   * **NOTE**: Since this is a theme bridge to *future* versions, the theme could have breaking changes.
+   */
+  bridgeToFutureVersions?: boolean;
```

Comment:
We should bikeshed this name a little bit 😅

Reply:
I like "theme bridge" and "future". We should avoid `use` prefix to avoid confusion with hooks.

I was also thinking we might have other "future" options added (in the future 😅). So what if added a new `future` prop which accepts an object containing a `themeBridge` property?

```jsx
<ThemeProvider future={{ themeBridge: true }}>
```